account_invoice_line_unit_price_untaxed/tests/test_account_move_line_untaxed.py

```diff
@@ -140,3 +140,41 @@ def test_invoice_with_product_company2(self):
         invoice_line = first(self.invoice.invoice_line_ids)
         self.assertEqual(invoice_line.price_unit, 115)
         self.assertEqual(invoice_line.price_unit_untaxed, 100)
+
+    def test_invoice_with_decimal_precision(self):
+        """
+        Set tax with price include False
+        Set the product price to 118.573
+        Set product price precision to 3
+        Check prices are equivalent (price_unit == price_unit_untaxed) with
+        same
+        """
+        self.tax2.price_include = False
```

Comment:
IMO you should also ensure that no currency are created. Even if you do a `new(...)` it's better to ensure the behaviour won't change.

Reply:
@acsonefho I don't remember why you said this :sweat_smile: